sycl/test/usm/findplatforms.hpp

```diff
@@ -11,24 +11,23 @@ bool findPlatformAndDevice(cl_device_type deviceType,
   cl_int errorCode;
 
   errorCode = clGetPlatformIDs(0, nullptr, &numPlatforms);
-  if (errorCode != CL_SUCCESS) return false;
```

Comment:
Why are you removing check. Does it make sense to continue if this call fails?

Reply:
Error checking was causing problems.  Those slipped out.